src/user-guide/writing-workflows/runtime.rst

```diff
@@ -837,10 +840,10 @@ Late Events
 
 .. warning::
 
-  The scheduler can only check for lateness once a task has appeared in its
-  active task window. In Cylc 8 this is usually when the task is actually
-  ready to run, which severely limits the usefulness of late events as
-  currently implemented.
+  The scheduler can only check for lateness once a task becomes
+  :term:`active <active task>`. In Cylc 8 this usually means the task
+  is ready, or nearly ready, to run, which limits the usefulness of late
+  events as currently implemented.
```

Comment:
Do we need the "as currently implemented"?

Reply:
You're right, we don't.